.github/workflows/build.yaml

```diff
@@ -16,6 +16,7 @@ jobs:
   build:
     name: Build
     runs-on: windows-latest
+    continue-on-error: true # Since remaining jobs are just cancelled on one job failure, we could allow them to finish and report the errors
```

Comment:
W.r.t my comment here https://github.com/microsoft/windows-drivers-rs/pull/306#discussion_r2113165257

IMO, continue-on-error is probably okay on this repo, assuming the number of changes that are made to this repo is relatively lesser compared to windows-drivers-rs repo. Having this setting enabled also allowed us to catch a recent issue that happened only using nightly toolchain.